crates/contracts/build.rs

```diff
@@ -523,11 +523,10 @@ fn main() {
     generate_contract("Trader");
     generate_contract("Solver");
 
-    // Support contract used for balance simulation.
+    // Support contract used for various order simulations.
```

Comment:
Otherwise it sounds like the comment should rather be above `SimulateCode` (instead of above a block of contracts).
```suggestion
    // Support contracts used for various order simulations.
```